action-server/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM rasa/rasa:1.9.4
 EXPOSE 8080
 
+COPY requirements.txt requirements.txt
```

Comment:
After having rebased on master, you'll have to change the `prepare_core_build` job in the circle-ci script: Within the `Prepare NLU training files` step, add `cp requirements.txt ${COMPONENT_PATH}`.

Reply:
done